examples/integration/BUILD

```diff
@@ -77,8 +98,10 @@ string_flag(
         watchos_simulator_cpus = simulator_cpu.removeprefix("sim_"),
         xcode_configurations = XCODE_CONFIGURATIONS,
     )
-    for simulator_cpu in [
-        "sim_arm64",
-        "x86_64",
+    for simulator_cpu, build_mode in [
+        ("sim_arm64", "xcode"),
+        ("sim_arm64", "bazel"),
+        ("x86_64", "xcode"),
+        ("x86_64", "bazel"),
```

Comment:
Can you make this change in its own PR to land before this?

Reply:
Absolutely, I suppose maybe CI has to be changed as well?

Reply:
No, it already does this for fixtures. the CI is blocked by you not being a regular contributor. I can unblock that.